docs/docs/api/isochrone/api-reference.md

```diff
@@ -65,20 +67,6 @@ When making a map, drawing the isochrone contours as lines is more straightforwa
 
 (TODO: write something about rendering the GeoTIFF output.)
 
-## Future work on the isochrone service
-
-The Isochrone service is in active development. To report software issues or suggest enhancements, open an issue in the [Valhalla GitHub repository](https://github.com/valhalla/valhalla/issues).
-
-Several other options are being considered as future service enhancements. These include:
-
-* ~~Using distance rather than time for each unit.~~
```

Comment:
Good catch! Nice to see those 7-8 year old TODOs were solved naturally over time:)